docs/tutorials/custom-metrics/one-version.md

```diff
@@ -37,7 +37,7 @@ Validate [SLOs](../../getting-started/concepts.md#service-level-objectives) for
 ```shell
 iter8 k launch \
 --set "tasks={custommetrics,assess}" \
---set custommetrics.templates.istio-prom="https://raw.githubusercontent.com/iter8-tools/iter8/v0.12.2-pre/testdata/custommetrics/istio-prom.tpl" \
+--set custommetrics.templates.istio-prom="https://raw.githubusercontent.com/iter8-tools/iter8/v0.12.0/testdata/custommetrics/istio-prom.tpl" \
```

Comment:
Fix the location of the template to the "correct" location!

Reply:
https://github.com/iter8-tools/docs/pull/48